sevntu-checks/pom.xml

```diff
@@ -480,7 +480,7 @@
                     <limit>
                       <counter>LINE</counter>
                       <value>COVEREDRATIO</value>
-                      <minimum>0.98</minimum>
+                      <minimum>0.97</minimum>
```

Comment:
The coverage is unchanged. Replacing `String result = null;` with `final String result;` reduces the size of the method, thus the covered percentage changes from 0.98 to 0.979